test/bench.cpp

```diff
@@ -1,90 +1,405 @@
 #include <lorina/bench.hpp>
+#include <lorina/lorina.hpp>
 #include <cstdio>
 #include <fstream>
 #include <string>
 #include <vector>
+#include <stdio.h>
+#include <stdbool.h>
+#include <list>
+#include <algorithm>
+
+#include <SDL.h>
+#include <SDL_ttf.h>
+
+// Define MAX and MIN macros
+#define MAX(X, Y) (((X) > (Y)) ? (X) : (Y))
+#define MIN(X, Y) (((X) < (Y)) ? (X) : (Y))
+
+// Define screen dimensions
+#define SCREEN_WIDTH    800
+#define SCREEN_HEIGHT   600
```

Comment:
Screen sizes varies widely and should be set manually on launch or be detected automatically.

Reply:
Magic numbers are not allowed, use enumerations or named constants instead.